docs/lib/graphqlapi/fragments/flutter/query-data.md

```diff
@@ -0,0 +1,58 @@
+## Query item
+
+Now that you were able to make a mutation, take the `id` that was printed out and use it in your query to retrieve data.
+
+```dart
+try {
+    String graphQLDocument = '''query GetTodo(\$id: ID!) {
+      getTodo(id: \$id) {
+        id
+        name
+        description
+      }
+    }''';
+
+    var operation = Amplify.API.query(
+        request: GraphQLRequest<String>(
+            document: graphQLDocument,
+            variables: {"id": "8e0dd2fc-2f4a-4dc4-b47f-2052eda10775"}));
```

Comment:
Is it possible to show how to retrieve this Id from the mutation result? Based on our example schema and selection set used in mutation, it should be consistent on how Id gets retrieved instead of using a dummy Id here.

Reply:
Since we only have advanced flows, and nowhere in our docs we are showing how to access the data, I think it's a huge value add here to show, and possibly in other examples as well (e.g. a query example)

Reply:
Added details for how to work with the mutation response as discussed.